app/entry.server.tsx

```diff
@@ -24,7 +24,7 @@ export default function handleRequest(
   remixContext: EntryContext,
   loadContext: AppLoadContext,
 ) {
-  return isbot(request.headers.get("user-agent"))
+  return isbot(request.headers.get("user-agent") || "")
```

Comment:
this should not be needed.

`Headers.get(name: string): string | null`
`isbot(userAgent?: string | null | undefined): boolean`

Reply:
Indeed it could be no needed, I just referenced this from the latest Remix Vite project sample and htought they're doing it for a reason